Find a functionally equivalent code:android.widget.CalendarView.getFirstDayOfWeek:COMMENT
Method Modifier: public      
Comment:/**
 * Gets the first day of week.
 *
 * @return The first day of the week conforming to the {@link CalendarView}
 * APIs.
 * @see Calendar#MONDAY
 * @see Calendar#TUESDAY
 * @see Calendar#WEDNESDAY
 * @see Calendar#THURSDAY
 * @see Calendar#FRIDAY
 * @see Calendar#SATURDAY
 * @see Calendar#SUNDAY
 *
 * @attr ref android.R.styleable#CalendarView_firstDayOfWeek
 */

@@ -1,3 +1,3 @@
 {
-    return mFirstDayOfWeek;
+    return mDelegate.getFirstDayOfWeek();
 }

